<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRaycastingSettings</class>
 <widget class="QDialog" name="DialogRaycastingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raycasting settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxJittered">
     <property name="text">
      <string>Enable jittered sampling</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBoxPhongShading">
     <property name="text">
      <string>Phong shading</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelStepSize">
       <property name="text">
        <string>Step size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderStepSize">
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="sliderPosition">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelstepSizeValue">
       <property name="text">
        <string>0.01</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxVariableStep">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Variable</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelInterpolation">
       <property name="text">
        <string>Interpolation </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonInterpolationNN">
       <property name="text">
        <string>NN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonInterpolationLinear">
       <property name="text">
        <string>Linear</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonInterpolationCubic">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cubic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
